test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -162,6 +162,19 @@ public void clear() throws Exception {
         assertTrue(isEmpty(defaultAddressBook.getAllTags()));
     }
 
+    @Test
+    public void sort() throws Exception {
```

Comment:
The test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`.
You can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.